config/config.go

```diff
@@ -18,6 +18,9 @@ var VersionNumber = StaticVersionNumber
 // WebServerPort is the port for Owncast's webserver that is used for this execution of the service.
 var WebServerPort = 8080
 
+// Bind WebServer to this IP address. Be secure by default.
+var WebServerIP = "127.0.0.1"
```

Comment:
While I totally understand why you'd want to default to the loopback address, most people running Owncast won't know about this flag, not to mention all the existing users. They'll hit their public interface and nothing will come back, and they won't know why.  So I suggest we listen on `0.0.0.0` by default so it will continue to work for everyone.